README.md

```diff
@@ -11,7 +11,9 @@ Easily integrate CSI Cameras into your Viam projects using this camera module.
 
 ### Supported Modules
 - **Nvidia Jetson**: Use the `csi-cam` module. [More Info](./docs/JETSON.md)
-- **Raspberry Pi**: Use the `csi-cam-pi` module. [More Info](./docs/PI.md)
+- **Raspberry Pi**:
+  -  For Raspberry Pi CSI camera v3.0, use the `csi-cam-pi` module. [More Info](./docs/PI.md)
+  -  For Raspberry Pi CSI cameras v1.3 and v2.0 you can [enable legacy mode](https://docs.viam.com/operate/reference/prepare/rpi-setup/#enable-communication-protocols) and use the [`webcam` model](https://docs.viam.com/operate/reference/components/camera/webcam/).
```

Comment:
Might be good to add "optionally" to legacy mode + webcam.  The csi-cam-pi model is intended to work with all official raspberry pi csi cams.